README.md

```diff
@@ -25,10 +25,22 @@ npm run coverage
 ```
 
 ## Deployment
-```
-npm run deployOP
-npm run deployFTM
-```
+1. In ```000_deploy_dest_chain.ts```, fill in the ```veIB``` address
+2. Execute:
+    ```bash
+    npx hardhat deploy --tags deploy-dest --network optimism
+    ```
+3. In ```001_deploy_src_chain.ts```, fill in the ```receiver``` address (veMigrationDest deployed in Step 2)
```

Comment:
get deployed contract address using `get`: https://github.com/ibdotxyz/ib-deployment/blob/op/deploy/007_set_comptroller.ts#L11

might need to check out how to get from different network